### PR TITLE
Update `Bancho`

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
what do you think about casually inserting a status.ppy.sh link here? seems like a fitting place

File: wiki/Bancho_(server)/en.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
   - infrastructure
 ---
 
-# Bancho (game server)
+# Bancho (server)
 
 *For other uses, see [Bancho (disambiguation)](/wiki/Disambiguation/Bancho).*
 
-**osu!bancho** is the server which handles user presence, chat and multiplayer services.
+**Bancho** (previously *osu!bancho*) is a code name of the server component which handles interactions between players running stable versions of osu!:
```
Comment (peppy) [inline wiki/Bancho_(server)/en.md:10]: Weird, I don't remember renaming it but I guess I did in the client in a hope to stop thinking it was what we called the "official server" 🙄

File: wiki/Bancho_(server)/en.md [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+
+- [Online presence](/wiki/Client/Interface/Chat_console#extended-chat-console) and the [IRC gateway for chat](/wiki/Community/Internet_Relay_Chat)
+- [Spectator](/wiki/Gameplay/Spectating) mode
+- [Multiplayer system](/wiki/Client/Interface/Multiplayer) and [tournament management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)
+
+During maintenance periods, Bancho is sometimes restarted. This is relayed to all online users via an in-game announcement system, as well as periodic warning messages in the chat with countdown. Bancho restarts do not affect score submission, since it is handled by different components.[^score-upload-ref]
```
Comment (peppy) [inline wiki/Bancho_(server)/en.md:16]:
```suggestion
During maintenance periods, Bancho is sometimes restarted. This is relayed to all online users via an in-game announcement system, as well as periodic warning messages in the chat with countdown. Bancho restarts only affect chat and multiplayer, as all other functionality is handled via different APIs. [^score-upload-ref]
```

File: wiki/Bancho_(server)/en.md (hunk 1 above)
Comment (peppy) [reply wiki/Bancho_(server)/en.md:10]:
```suggestion
**Bancho** (previously *osu!bancho*) is a code name of the server component which handles real-time interactions between players running stable versions of osu!:
```

File: wiki/Bancho_(server)/en.md [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
+# Bancho (server)
 
 *For other uses, see [Bancho (disambiguation)](/wiki/Disambiguation/Bancho).*
 
-**osu!bancho** is the server which handles user presence, chat and multiplayer services.
+**Bancho** (previously *osu!bancho*) is a code name of the server component which handles interactions between players running stable versions of osu!:
+
+- [Online presence](/wiki/Client/Interface/Chat_console#extended-chat-console) and the [IRC gateway for chat](/wiki/Community/Internet_Relay_Chat)
+- [Spectator](/wiki/Gameplay/Spectating) mode
+- [Multiplayer system](/wiki/Client/Interface/Multiplayer) and [tournament management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)
```
Comment (peppy) [inline wiki/Bancho_(server)/en.md:14]: Probably mention chat? Not just the "IRC gateway". 
It's the number one primary purpose of bancho after all.

File: wiki/Bancho_(server)/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Bancho_(server)/en.md]: I rechecked and it's called `osu!Bancho` in the chat welcome message, so you didn't, correct

File: wiki/Bancho_(server)/en.md (hunk 3 above)
Comment (TicClick) [reply wiki/Bancho_(server)/en.md:14]: added to the first item

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**